website/docs/advanced/override_grammar/basic.md

```diff
@@ -125,7 +126,17 @@ To include a single quote in a single quoted string escape it : `\'`. Same for d
 </div>
 </div>
 
- 
+### Unquoted values
+Override values can contain whitespaces and other special characters.
+Some examples of legal values: `hello world`,`   hello world  `, `https://${server}:${env:PORT}`.
+Unquoted whitespaces are allowed in the middle of the word, while leading and trailing whitespaces are stripped.
+Normally, your shell will interpret values with whitespaces as being multiple parameters (`key=a b` would be interpreted as `key=a` and `b`).
+To prevent this you can quote them with a single quote. For example:
+
+```bash
+$ python my_app.py 'msg=hello world'
+```
+
```

Comment:
What do you think about (not suggestion because GitHub does not render it correctly).

### Whitespaces in unquoted values
Unquoted Override values can contain non leading or trailing whitespaces.
For example, `msg=hello world` is a legal override (key is `msg` and value is the string `hello world`).
Normally, your shell will interpret values with whitespaces as being multiple parameters (`key=a b` would be interpreted as `key=a` and `b`).
To prevent this you can quote them with a single quote. For example:

```bash
$ python my_app.py 'msg=hello world'
```

Note that trailing and leading whitespace are ignored, the above is equivalent to:
```bash
$ python my_app.py 'msg=    hello world    '
```

Reply:
Looks good to me, I pushed it in b54615cc